ci.ts

```diff
@@ -1,6 +1,37 @@
-
 const testSuiteId: string = process.params.testSuiteId
-const token: string = process.env.AUTOMATOR_TOKEN
+const key_id: string = process.env.AUTOMATOR_USER_KEY_ID
```

Comment:
you can remove the typescript file, it's mostly an example not used by anyone currently. we can focus on the shell scripts for now until we have a better plan